pkg/workflows/create.go

```diff
@@ -265,6 +265,12 @@ func (s *InstallEksaComponentsTask) Run(ctx context.Context, commandContext *tas
 			commandContext.SetError(err)
 			return &CollectDiagnosticsTask{}
 		}
+		logger.Info("Installing EKS-D components on workload cluster")
+		err = commandContext.ClusterManager.InstallEksdComponents(ctx, commandContext.ClusterSpec, commandContext.WorkloadCluster)
```

Comment:
If we move the upgrade method maybe we should move this method to something like `eksd.Installer`